switchmap/server/db/ingest/ingest.py

```diff
@@ -536,10 +535,10 @@ def insert_macips(items, test=False):
 
     # Insert the values
     if bool(test) is False:
-        _macip.insert_row(rows)
+        _macip.bulk_insert_rows(rows)
     else:
-        for row in sorted(rows, key=attrgetter("idx_mac", "idx_ip")):
-            _macip.insert_row(row)
+        row = sorted(rows, key=attrgetter("idx_mac", "idx_ip"))
+        _macip.bulk_insert_rows(row)
```

Comment:
Use `insert_rows`. The FOR loop was wasteful. Thanks for catching that.

1. Update `db.add_all` to use the new `bulk_insert_rows` method
4. Apply this to all the` switchmap/server/db/table` files. It's simpler. There v2.0 of sqlalchemy has a `bulk_insert_rows` replacement.